docs/amber_script.md

```diff
@@ -197,9 +197,10 @@ TODO(dsinclair): Sync the BufferTypes with the list of Vulkan Descriptor types.
 ```
 
 ```
-  # Attach a vertex data buffer to the pipeline
-  VERTEX_DATA <buffer_name>
-  # Attach an index data buffer to the pipeline
+  # Set |buffer_name| as the vertex data at location |val|.
```

Comment:
Is the IDX val the shader's input Location attribute for the vertex data.
(That is, this is for populating VkVertexInputAttributeDescription::location )

So I think it should use keyword LOCATION instead of IDX.

Reply:
done

Reply:
done